### ISSUE TITLE
[Feature Request] Forum Accounts - "Post categories" gets content from non-owner posts

### ISSUE THREAD

**Issue by WNYmathGuy** [friendica/friendica#6899]
### Expected behavior
1. A non-owner makes a post to a Forum account's page with Categories and/or Hashtags.
1. The new items posted by the non-owner user get aggregated to the tag-cloud area and the category list area.

### Actual behavior
1. A non-owner makes a post to a Forum account's page with Categories and/or Hashtags.
1. Only tag items posted by the non-owner user get aggregated to the tag-cloud area, and the category list area gets no new content.

### Steps to reproduce the problem
It's this way by design.

### Friendica version you encountered the problem
2019.01

### Friendica source (git, zip)
git

### PHP version
7.2.15

### SQL version
MariaDB 10.1.38

<hr />

#### Categories are activated

![Screenshot from 2019-03-17 16-53-49](https://user-images.githubusercontent.com/28640985/54498358-c19fef80-48dc-11e9-8777-52479e03a802.png)

<hr />

#### Tag cloud is activated

![Screenshot from 2019-03-17 16-53-57](https://user-images.githubusercontent.com/28640985/54498377-f613ab80-48dc-11e9-8312-c0ac790bff40.png)

<hr />

#### normal user account with complete category and tag cloud content

![Screenshot from 2019-03-17 16-54-59](https://user-images.githubusercontent.com/28640985/54498396-22c7c300-48dd-11e9-82e5-201f356e561c.png)

<hr />

#### Forum with non-owner posts including tags and categories where new categories never propagate to the Forums area for them

![Screenshot from 2019-03-17 17-27-30](https://user-images.githubusercontent.com/28640985/54498407-4428af00-48dd-11e9-93cb-a1aed348c142.png)

<hr />

### Spitballing here ;)
1. What about subcategories?
1. What about a choser menu appearing in the Categories entry textfield to make reuse easier and spelling errors harder to make?

**Comment by tobiasd** [comment]
This is because categories is a thing for sorting your postings on your wall only, that are not distributed to your contacts. A forum is a remote contact, so it does not see the category of a posting. Which is only true for postings you write on your node with the `!` notation to mention the forum.

A different situation would be if you remote-auth on the forum account. When you post a new top-level posting it mit be possible to set the category for the posting. I've never tried to do so.

**Comment by WNYmathGuy** [comment]
> This is because categories is a thing for sorting your postings on your wall only, that are not distributed to your contacts. A forum is a remote contact, so it does not see the category of a posting. Which is only true for postings you write on your node with the `!` notation to mention the forum.
> 
> A different situation would be if you remote-auth on the forum account. When you post a new top-level posting it mit be possible to set the category for the posting. I've never tried to do so.

@tobiasd I, _as a non-technical user_, had no way to assume all that. As a casual user, I thought the Categories were visible on my ordinary account to visitors such that they could filter my posts to look at what they wanted to drill down to. It seemed logical that if a Forum account was a shared resource that the categories tool should have extended to it to benefit the group of shared resource users.

**Comment by tobiasd** [comment]
> @tobiasd I, _as a non-technical user_, had no way to assume all that.
> As a casual user, I thought the Categories were visible on my
> ordinary account to visitors such that they could filter my posts to
> look at what they wanted to drill down to.

Yes exactly. This is what categories are doing. The important thing is, that categories are not transmitted to your contacts.

> It seemed logical that if a Forum account was a shared resource that the categories tool should have extended to it to benefit the group of shared resource users.

Absolutely understandable :) and yet I think you are the first one raising this request.

I have checked the "what if you remote-auth at a forum" option. And currently the situation is, that you cannot set a category for the posting you write remote to the forum either. So this is definitively something currently not possible, but worth exploring I think,
